worlds/witness/test/test_disable_non_randomized.py

```diff
@@ -0,0 +1,36 @@
+from ..rules import _has_lasers
+from ..test import WitnessTestBase
+
+
+class TestDisableNonRandomized(WitnessTestBase):
+    options = {
+        "disable_non_randomized_puzzles": True,
+        "shuffle_doors": "panels",
+        "early_symbol_item": False,
+    }
+
+    def test_unrandomized_locations_do_not_exist(self) -> None:
```

Comment:
Did you miss this one?

Reply:
I probably missed a bunch of stuff in this rework, let me go over it again today